fictadvisor-api/src/v2/api/services/ScheduleService.ts

```diff
@@ -584,7 +566,39 @@ export class ScheduleService {
     });
   }
 
-  async updateDiscipline (
+  private getNewDateTime (event: DbEvent, {
+    startTime,
+    endTime,
+    changeStartDate,
+    changeEndDate,
+    period,
+  }: UpdateEventDTO) {
+    if (!startTime && changeStartDate) throw new ObjectIsRequiredException('startTime');
+    if (!endTime && changeEndDate) throw new ObjectIsRequiredException('endTime');
```

Comment:
As you have added new exception it should be documented in `ApiBadRequestResponse` decorator of update method in `ScheduleController`